asto/asto-core/src/main/java/com/artipie/asto/Merging.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2023 artipie.com
+ * https://github.com/artipie/artipie/blob/master/LICENSE.txt
+ */
+package com.artipie.asto;
+
+import io.reactivex.Flowable;
+import io.reactivex.Maybe;
+import java.nio.ByteBuffer;
+
+/**
+ * Merges ByteBuffer objects to bigger ones, according to specified `size`.
+ * Produces new ByteBuffer blocks according to specified target size.
```

Comment:
It's impossible to understand from the description, what are we going to get?
What does `size` present `min` or `max` size of a `ByteBuffer`?